core/modules/user/config/user.role.authenticated.json

```diff
@@ -2,6 +2,8 @@
     "_config_name": "user.role.authenticated",
     "name": "authenticated",
     "label": "Authenticated",
+    "admin_description": "Special role assigned to all accounts. Permissions assigned to this role are given to all other roles except the anonymous role.",
+    "description": "Automatically assigned role to all accounts.",
```

Comment:
I would either lose the word "role" altogether (it is mentioned in the field label when assigning roles, so it's redundant), or move it to the beginning of the sentence.